src/OrchardCore/OrchardCore.ContentManagement.GraphQL/Queries/ContentItemsFieldType.cs

```diff
@@ -64,9 +65,9 @@ public ContentItemsFieldType(string contentItemName, ISchema schema, IOptions<Gr
             _defaultNumberOfItems = settingsAccessor.Value.DefaultNumberOfResults;
         }
 
-        private async Task<IEnumerable<ContentItem>> Resolve(ResolveFieldContext context)
+        private async Task<IEnumerable<ContentItem>> Resolve(IResolveFieldContext context)
         {
-            var graphContext = (GraphQLContext)context.UserContext;
+            var graphContext = (GraphQLUserContext)context.UserContext;
```

Comment:
hink this doesnt need to be created and passed down the stack, the context is passed anyway and has the RequestServices on it